components/httpd_server/src/httpd/server.cpp

```diff
@@ -0,0 +1,185 @@
+#include "server.hpp"
+#include "esp_log.h"
+
+namespace httpd
+{
+
+  Server::Server() : Server(80)
```

Comment:
Please keep the constructors together. Between the constructors is a destructor now.